Mlem/Views/Tabs/Search/BubblePicker.swift

```diff
@@ -5,67 +5,88 @@
 //  Created by Sjmarf on 18/09/2023.
 //
 
-import SwiftUI
 import Dependencies
+import SwiftUI
+
+enum DividerPlacement {
+    case top, bottom
+}
 
 struct BubblePicker<Value: Identifiable & Equatable & Hashable>: View {
     @Dependency(\.hapticManager) var hapticManager
     
     @Binding var selected: Value
     let tabs: [Value]
+    let dividers: [DividerPlacement]
```

Comment:
This should really be a `Set`, though I now realise I'm also guilty of using arrays where I shouldn't for `UserListRow` complications 😅